components/Dialogs/tests/snapshot/MDCAlertControllerLocalizationTests.m

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
+
+#import "MaterialDialogs.h"
+
+static NSString *const kTitleUrdu = @"عنوان";
+static NSString *const kMessageUrdu =
+    @"براہ کرم اپنا نیٹ ورک کنکشن چیک کریں اور دوبارہ کوشش کریں۔براہ کرم اپنا نیٹ ورک کنکشن چیک "
+    @"کریں اور دوبارہ کوشش کریں۔";
+
+static NSString *const kActionMediumUrdu = @"درمیانی";
+static NSString *const kActionLowUrdu = @"کم";
+
+@interface MDCAlertControllerLocalizationSnapshotTests : MDCSnapshotTestCase
```

Comment:
Nit: Please add header docs to this class and the properties.

Reply:
Addressed in 9ed4cbe40464c1fa6c7f23b34948efd3d7a61fbc.